core/src/main/groovy/com/novoda/gradle/release/AndroidArtifacts.groovy

```diff
@@ -1,42 +1,51 @@
 package com.novoda.gradle.release
 
 import org.gradle.api.Project
+import org.gradle.api.Task
+import org.gradle.api.component.SoftwareComponent
 import org.gradle.api.tasks.bundling.Jar
 import org.gradle.api.tasks.javadoc.Javadoc
 
 class AndroidArtifacts implements Artifacts {
 
     def variant
 
+    // TODO: Declare that variant is
+    // https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.LibraryExtension.html#com.android.build.gradle.LibraryExtension:libraryVariants
     AndroidArtifacts(variant) {
         this.variant = variant
     }
 
+    @Override
     def all(String publicationName, Project project) {
         [sourcesJar(project), javadocJar(project), mainJar(project)]
     }
 
     def sourcesJar(Project project) {
-        project.task(variant.name + 'AndroidSourcesJar', type: Jar) {
-            classifier = 'sources'
+        String taskName = variant.name + 'AndroidSourcesJar'
+        project.tasks.create(taskName, Jar) {
+            it.classifier = 'sources'
```

Comment:
it is not required

Reply:
Why isn't it required? 🤔 
I haven't changed here something. Just formatted the code.
Can you pls explain what and why it isn't required?!

Reply:
`it` wasn't there before https://github.com/novoda/bintray-release/pull/193/files?diff=unified#diff-e5cce508cbb1140eb99d9d210c874104L21

Groovy magic ✨, the Task is the delegate of the closure and therefore you can access the properties of it.

Reply:
For ~me~ the IDE the `it` is required:
<img width="354" alt="bildschirmfoto 2018-02-19 um 20 03 17" src="https://user-images.githubusercontent.com/10229883/36393481-1a171492-15b0-11e8-9aa1-8a4c2656ef01.png">
<img width="246" alt="bildschirmfoto 2018-02-19 um 20 03 32" src="https://user-images.githubusercontent.com/10229883/36393482-1a322188-15b0-11e8-8e37-ee2fd2c4488e.png">